docs/pages/access-controls/sso.mdx

```diff
@@ -398,6 +402,13 @@ version: v2
 </TabItem>
 </Tabs>
 
+<Admonition type="note" title="IdP-initiated SSO">
+  Enabling the `allow_idp_initiated` flag in SAML connectors allows users to
+  login to Teleport with one click from the dashboard provided by the IdP.
+  This comes with [multiple security considerations worth reviewing](https://github.com/gravitational/teleport/blob/master/rfd/0073-idp-initiated-login.md).
```

Comment:
RFDs are intended for a very technical Teleport-developer audience, while product documentation should be more geared towards the average user.

Do you think we can highlight the major security concerns here without blowing up the page instead of linking to an RFD?

Reply:
I added this as an admonition for this reason. Although I can briefly list the security concerns in the form of the headlines used in the RFD, I don't believe that information alone would be useful to a Teleport cluster administrator and help them make an informed decision. I feel like as an admin I would want more detailed information to make a decision. I did consider writing a separate page for this but it ended up being very similar to the RFD itself.

Reply:
Let's see what @ptgott thinks when he gets back from vacation. I'm still leaning towards _not_ linking to the RFD here, but I'm happy to be told I'm wrong.

Reply:
First, I think we should move this Admonition into the appropriate `TabItem`s in the `Tabs` above so OIDC/GitHub users don't need to see irrelevant information.

If a reader needs to be familiar with a complex list of security considerations before enabling `allow_idp_initiated`, I don't think we can count on readers finding this `Admonition`, which is somewhat buried here. Instead, we can write a quick guide to IdP-initiated logins that spells out these considerations for Teleport's audience (i.e., paraphrasing but not linking to the RFD). We can then remove the mentions of `allow_idp_initiated` from elsewhere in the docs so readers don't accidentally enable it.

If a reader doesn't need to be familiar with all the considerations for using IdP-initiated logins, but only needs to know that it's generally unsafe, I think we can include one or two headline issues in this Admonition and remove the link. We'll also want to include similar Admonitions in our individual SSO guides, since a user might be arriving there from an organic search rather than from this guide.

Reply:
@ptgott I think the other solution works well. I've added per-tab admonitions and also per-page admonitions for relevant connector guide pages. Can you give this another look?